backends/advanced/src/advanced_omi_backend/services/transcription/__init__.py

```
@@ -1,120 +1,269 @@
+        if "config" in start_msg and isinstance(start_msg["config"], dict):
+            start_msg["config"].setdefault("sample_rate", sample_rate)
+            if diarize:
+                start_msg["config"]["diarize"] = True
+
+        ws = await websockets.connect(url, open_timeout=10)
+        await ws.send(json.dumps(start_msg))
+        # Wait for confirmation; non-fatal if not provided
+        try:
+            await asyncio.wait_for(ws.recv(), timeout=2.0)
+        except Exception:
+            pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)